model/src/main/kotlin/config/AnalyzerConfiguration.kt

```diff
@@ -19,19 +20,22 @@
 
 package org.ossreviewtoolkit.model.config
 
+import com.fasterxml.jackson.annotation.JsonAlias
```

Comment:
Commit message:

> In order to make the AnalyzerConfiguration deserialisable, default values are set.

The reason is not that the class is generally not deserializable without the default values, but that you want to be able to omit some of the values in the config file, especially with the SW360 configuration you are adding in the next commit.

Reply:
This import should not be added in this commit.